File: app/src/main/java/bj/discogsbrowser/model/collection/AddToCollectionResponse.java

package bj.discogsbrowser.model.collection;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AddToCollectionResponse
{

    @SerializedName("instance_id")
    @Expose
    private String instanceId;
    @SerializedName("resource_url")
    @Expose
    private String resourceUrl;

    public String getInstanceId()
    {
        return instanceId;
    }

    public void setInstanceId(String instanceId)
    {
        this.instanceId = instanceId;
    }

    public String getResourceUrl()
    {
        return resourceUrl;
    }

    public void setResourceUrl(String resourceUrl)
    {
        this.resourceUrl = resourceUrl;
    }

}

File: app/src/main/java/bj/vinylbrowser/model/collection/AddToCollectionResponse.kt
package bj.vinylbrowser.model.collection

import com.google.gson.annotations.SerializedName

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class AddToCollectionResponse(@SerializedName("instance_id") var instanceId: String = "",
                                   @SerializedName("resource_url") var resouceUrl: String = "")